src/views/Editor.vue

```diff
@@ -310,14 +336,15 @@ export default {
 					await apiClient.put("/datasets/" + currentId, payload)
 					const { data } = await apiClient.get(`/datasets/${currentId}`)
 					this.qvainData = data
+					this.clearRecord()
+					this.openRecord(this.id)
```

Comment:
I like to save often and the editor reload and loss of UI state (scroll position, open tabs) this causes is somewhat disorienting. What does adding these fix?

Reply:
hmm

Reply:
i think that this has not been done on purpose

Reply:
this has been reverted